pages.fr/common/az.md

```diff
@@ -2,7 +2,7 @@
 
 > La CLI officielle de Microsoft Azure.
 > Certaines commandes comme `az login` ont leur propre documentation.
-> Plus d'informations : <https://learn.microsoft.com/cli/azure>.
+> Plus d'informations : <https://learn.microsoft.com/cli/azure/reference-index>.
```

Comment:
Question for other reviewers: should the doc links stay the same accross all translations, or should they be localized when a translation exist?
Here the french translation is: https://learn.microsoft.com/fr-fr/cli/azure/reference-index

Reply:
Thanks.